python/python/glide/async_commands/core.py

```diff
@@ -1496,6 +1496,94 @@ async def linsert(
             ),
         )
 
+    async def lmove(
+        self,
+        source: str,
+        destination: str,
+        wherefrom: ListDirection,
+        whereto: ListDirection,
+    ) -> Optional[str]:
+        """
+        Atomically pops and removes the left/right-most element to the list stored at `source`
+        depending on `wherefrom`, and pushes the element at the first/last element of the list
+        stored at `destination` depending on `whereto`.
+
```

Comment:
Please add cross slot notice and since secion

Reply:
https://github.com/aws/glide-for-redis/blob/e5992cf9000453b9a8911fbf2036f9112cc53ef5/java/client/src/main/java/glide/api/commands/ListBaseCommands.java#L507-L519